docs/markdown/authoritative/migration.md

```diff
@@ -55,3 +55,30 @@ Parse only this zone file. Conflicts with `--named-conf` parameter.
 
 ## `--zone-name`
 When parsing a single zone without $ORIGIN statement, set this as the zone name.
+
+# Migrating from backend to another
+
+NB! This is experimental feature.
+
+Syntax: `pdnssec b2b-migrate old new`
+
+This tool lets you migrate data from one backend to another, it moves all data, including zones, metadata and crypto keys (if present). Some example use cases are moving from Bind style zonefiles to SQL based, or other way around, or moving from MyDNS to gMySQL.
+
+## Prerequisites
+
+ - Target backend must support everything the source supports
```

Comment:
That page contains a table of all the supported features per backend, so linking it makes the reader's job easier.